gemsiv/delaunay/quadedge.C

```diff
@@ -1,4 +1,9 @@
 #include "quadedge.h"
+#include "../../dummy.h"
```

Comment:
Ah, thanks. Yes, I like fakegl, and please add a comment at the top explaining why this file was added. And thanks very much for bringing this code into this century!